tests/integration/targets/wafv2_ip_set/tasks/main.yml

```diff
@@ -0,0 +1,227 @@
+---
+- module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+
+  block:
+    - name: set up aws connection info
+      set_fact:
+        aws_connection_info: &aws_connection_info
```

Comment:
Is this here for the 2.9 tests?  You can open a PR directly against the stable-2.9 branch for mod_defaults, like this: https://github.com/ansible/ansible/pull/72145

Reply:
Hm, yes. Do we need to backport it to ansible 2.9?

Reply:
Ok I reverse the `&aws_connection_info` construct.  
Locally everything pass.